deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -76,14 +76,21 @@ export class MigrationConsoleStack extends MigrationServiceCore {
                 "kafka-cluster:*"
             ]
         })
-
+        const allReplayerServiceArn = `arn:aws:ecs:${props.env?.region}:${props.env?.account}:service/migration-${props.stage}-ecs-cluster/migration-${props.stage}-traffic-replayer*`
+        const updateReplayerServicePolicy = new PolicyStatement({
+            effect: Effect.ALLOW,
+            resources: [allReplayerServiceArn],
+            actions: [
+                "ecs:UpdateService"
```

Comment:
This reads weird.  The replayers can update themselves?

Reply:
Only the migration console can update the replayers